src/com/github/The127/MScript/MScriptCompiler.java

```diff
@@ -13,7 +13,11 @@
 
 public class MScriptCompiler {
 	public static void main(String[] args) throws IOException {
-		var code = new String(Files.readAllBytes(Paths.get("res/test.ms")));
+		if(args.length < 1){
+			System.out.println("Missing file argument.");
+			System.exit(1);
+		}
+		var code = new String(Files.readAllBytes(Paths.get(args[0])));
```

Comment:
Throws either `IOException` or `InvalidPathException`.

Maybe converte them to user friendly error messages in the future.